src/modules/hub.bicep

```diff
@@ -86,7 +82,7 @@ module dataFactory 'Microsoft.DataFactory/factories/deploy.bicep' = {
 }
 
 resource storageAccountLookup 'Microsoft.Storage/storageAccounts@2021-06-01' existing = {
-  name: storageAccount.name
+  name: storageAccountName
```

Comment:
This fixes a bug that was causing the deployment to fail. `storageAccount.name` references the deployment name, not the storage account name.